gwcli/ceph.py

```diff
@@ -285,15 +287,18 @@ class RadosPool(UINode):
     display_attributes = ["name", "commit", "overcommit_PCT", "max_bytes",
                           "used_bytes", "type", "desc"]
 
-    def __init__(self, parent, pool_name, pool_md):
+    def __init__(self, parent, pool_name, pool_md, ec_profiles):
         UINode.__init__(self, pool_name, parent)
         self.pool_md = pool_md
```

Comment:
I'd drop `self.pool_md` -- it is unused.

Reply:
Done.